cc/main.cc

```diff
@@ -195,7 +202,8 @@ void WriteSgf(const std::string& output_dir, const std::string& output_name,
     }
   }
 
-  std::string player_name(file::Basename(FLAGS_model));
+  std::string player_name =
+      static_cast<std::string>(file::Basename(FLAGS_model));
```

Comment:
These are reverted in PR #452.

CLion marks these lines with squiggles, and it seemed to make sense because 'string_view::operator std::string()' is explicit. I didn't realize that this still allows direct initialization (but not copy initialization) without an explicit cast.